src/index.html

```diff
@@ -17,11 +30,354 @@
+          </h1>
+          <p class="card_product_code">Product code: 195434</p>
+          <div class="card_reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card_review">Reviews: 5</span>
+          </div>
+          <div class="card_product_price">
+            <span class="card_text_price">Price:</span>
+            <span class="card_price">$2,199</span>
+          </div>
+          <div class="card_button">
+            <button class="card_button_buy">BUY</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card_image"
+          src="images/imac.jpeg"
+          alt="Imagem IMAC"
+        />
+        <div class="card_content">
+          <h1 class="card_title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <p class="card_product_code">Product code: 195434</p>
+          <div class="card_reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card_review">Reviews: 5</span>
+          </div>
+          <div class="card_product_price">
+            <span class="card_text_price">Price:</span>
+            <span class="card_price">$2,199</span>
+          </div>
+          <div class="card_button">
+            <button class="card_button_buy">BUY</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card_image"
+          src="images/imac.jpeg"
+          alt="Imagem IMAC"
+        />
+        <div class="card_content">
+          <h1 class="card_title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+          <p class="card_product_code">Product code: 195434</p>
+          <div class="card_reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card_review">Reviews: 5</span>
+          </div>
+          <div class="card_product_price">
+            <span class="card_text_price">Price:</span>
+            <span class="card_price">$2,199</span>
+          </div>
+          <div class="card_button">
+            <button class="card_button_buy">BUY</button>
+          </div>
+        </div>
+      </div>
```

Comment:
The card elements are repeated multiple times with the same content. Consider using a template engine or JavaScript to dynamically generate these cards to avoid redundancy and make the code easier to maintain.